layersvt/vk_layer_settings.txt

```diff
@@ -1,152 +1,204 @@
+# Show Timestamp
+# =====================
+# <LayerIdentifier>.show_timestamp
+# Show the timestamp of function calls since start in microseconds
+lunarg_api_dump.show_timestamp = false
 
-# VK_LAYER_LUNARG_device_simulation Settings
-lunarg_device_simulation.filename = 
-lunarg_device_simulation.debug_enable = 0
-lunarg_device_simulation.exit_on_error = 0
+# Show Shader
+# =====================
+# <LayerIdentifier>.show_shader
+# Dump the shader binary code in pCode
+lunarg_api_dump.show_shader = false
 
-################################################################################
-#  VK_LAYER_LUNARG_screenshot Settings:
-#  ====================================
-#
-#    FRAMES:
-#    =======
-#    <LayerIdentifer>.frames : Comma separated list of frames to output as
-#    screen shots or a range of frames with a start, count, and optional
-#    interval separated by a dash. Setting the variable to \"all\" will output
-#    every frame. Example: \"5-8-2\" will output frame 5, continue until frame 13,
-#    dumping every other frame. Example: \"3,8-2\" will output frames 3, 8, and 9.
-#
-#    DIR:
-#    ====
-#    <LayerIdentifer>.dir : This can be set to specify the directory in which to
-#    create the screenshot files.
-#
-#    FORMAT:
-#    =======
-#    <LayerIdentifer>.format : This can be set to a color space for the output.
-#    It must be one of the following values: "UNORM", "SNORM", "USCALED", "SSCALED",
-#    "UINT", "SINT", "SRGB" or "USE_SWAPCHAIN_COLORSPACE".
-
-# VK_LAYER_LUNARG_screenshot Settings
-lunarg_screenshot.frames = 0-0
+# Show Parameter Details
+# =====================
+# <LayerIdentifier>.detailed
+# Dump parameter details in addition to API calls
+lunarg_api_dump.detailed = true
+
+# Hide Addresses
+# =====================
+# <LayerIdentifier>.no_addr
+# Dump "address" in place of hex addresses
+lunarg_api_dump.no_addr = false
+
+# Use Spaces
+# =====================
+# <LayerIdentifier>.use_spaces
+# Setting this to true causes all tab characters to be replaced with spaces
+lunarg_api_dump.use_spaces = true
+
+# Indent Size
+# =====================
+# <LayerIdentifier>.indent_size
+# Specifies the number of spaces that a tab is equal to
+lunarg_api_dump.indent_size = 4
+
+# Show Thread and Frame
+# =====================
+# <LayerIdentifier>.show_thread_and_frame
+# Show the thread and frame of each function called
+lunarg_api_dump.show_thread_and_frame = true
+
+
+# VK_LAYER_LUNARG_screenshot
+
+# Frames
+# =====================
+# <LayerIdentifier>.frames
+# Specifies list of frames to output as screenshots. It is specified as a comma
+# separated list of frames or a range of frames with a start, count, and
+# optional interval separated by a dash. Setting the variable to "all" will
+# output every frame. Example: "5-8-2" will output frame 5, continue until frame
+# 13, dumping every other frame. Example: "3,8-2" will output frames 3, 8, and
+# 9.
+lunarg_screenshot.frames = 
```

Comment:
The above description does not indicate that an empty string is a valid value, or how it will be interpreted.

Reply:
Again, this wording came from screenshot_layer.md. I had to go to the screenshot source code to see what it does if lunarg_screenshot.frames or VK_SCREENSHOT_FRAMES is null or empty. No screenshot it take if that is the case. I'll add a sentence for the unset/empty case.